interni/studio_fattibilita/main.tex

```diff
@@ -230,6 +230,55 @@
 		\newpage
 		%! AGGIUNGERE RESTANTI CAPITOLATI E CONCLUSIONE FINALE
 		
+		\subsection{C3 - GDP}
+		\subsubsection{Informazioni generali}
+		\begin{itemize}
+			\item \textbf{Titolo:} GDP;
+			\item \textbf{Sottotitolo:} Gathering Detection Platform;
+			\item \textbf{Proponente:} Sync Lab;
+			\item \textbf{Committente:} Prof. Tullio Vardanega e Prof. Riccardo Cardin.
+		\end{itemize}
+		
+		\subsubsection{Descrizione}
+		In seguito alla pandemia del virus COVID-19 scoppiata negli ultimi mesi sono sorti numerosi problemi. Dopo un periodo di quarantena forzata la vita doveva ripartire ed ai cittadini è stata permessa la circolazione. Durante la vita quotidiana si generano spesso assembramenti in numerosi luoghi e per non aggravare la situazione bisogna cercare di prevenire questo problema. Grazie all’informatica possiamo quindi scongiurare l’aggravarsi della situazione dovuta ai problemi sociali legati all’affollamento.
+		
+		\subsubsection{Finalità}
+		
+		Si vuole creare un’applicazione web che permetta di:
+		\begin{itemize}		
+		\item \textsc Misurare tramite dei contapersone e date le immagini/stream delle videocamere di contare le persone a bordo dei mezzi pubblici
+		\item \textsc Poter acquisire continuativamente nel tempo e in modalità a bassa latenza delle informazioni raccolte da dispositivi di flussi dati	
+		\item \textsc Elaborare in tempo reale i dati acquisiti per rappresentare le variazioni nel tempo dei dati monitorati, come ad esempio la variazione dei flussi delle persone in ambienti urbani. Confrontare e correlare tra loro dati provenienti da flussi diversi, ad esempio correlando i dati raccolti a bordo dei mezzi di trasporto di interesse con i dati raccolti in prossimità delle aree di interesse quali ad esempio fermate di autobus, strade dedicate al traffico veicolare, zone urbane. E' poi fondamentale archiviare tutti i dati acquisiti, ed i risultati delle loro elaborazioni affinché siano conservati nel tempo e comunque disponibili in qualunque caso sia richiesto il loro utilizzo. 
+		\item \textsc Identificazione, a partire dai dati acquisiti ed elaborati, di eventi che nel tempo sono risultati aver concorso ad esempio all’insorgere di alterazioni significative del flusso di utenti;
+		\item \textsc Previsione dell’insorgenza futura di variazioni significative di flussi di persone al fine di poter fornire degli indicatori automatici in grado di fornire supporto alle decisioni da parte del personale tecnico specializzato preposto a tale scopo.
+		\end{itemize}
+		 	
+		\subsubsection{Tecnologie}
+		Ci sono alcune scelte preferenziali per lo svolgimento del progetto:
+		\begin{itemize}		
+		\item \textsc Utilizzo di \glock{Java} e \glock{Angular} per lo sviluppo delle parti di \glock{Back-end} e \glock{Front-end} della componente Application del sistema
+		\item \textsc Il framework \glock{Leaflet} (https://leafletjs.com) per la gestione delle mappe (heatmap)

```

Comment:
Ogni `\item` in ogni elenco finisce con un punto e virgola a meno che non sia l'ultimo `\item` dell'elenco (nel qual caso termina con un punto).

Inoltre, ogni `\item` in ogni elenco comincia con la lettera minuscola tranne quando l'elenco è inizio di frase; in tal caso il primo `\item` ha lettera maiuscola (es. la 'L' maiuscola in riga 274 è corretta). Ovviamente il discorso non vale con i nomi propri.

```suggestion
		\item utilizzo di \glock{Java} e \glock{Angular} per lo sviluppo delle parti di \glock{Back-end} e \glock{Front-end} della componente Application del sistema;
		\item il framework \glock{Leaflet} (https://leafletjs.com) per la gestione delle mappe (heatmap);
```